extensions/credentials/identity-hub-credentials-jwt/src/test/java/org/eclipse/edc/identityhub/credentials/jwt/JwtCredentialEnvelopeTest.java

```diff
@@ -51,11 +51,11 @@ void verifyToVerifiableCredential() {
         var envelope = new JwtCredentialEnvelope(jwt);
 
         // Act
-        var result = envelope.toVerifiableCredential(OBJECT_MAPPER);
+        var result = envelope.toVerifiableCredentials(OBJECT_MAPPER);
 
         // Assert
         assertThat(result.succeeded()).isTrue();
-        var vc = result.getContent();
+        var vc = result.getContent().get(0);
```

Comment:
Please asset size before check the object:
```
assertThat(result.getContent()).hasSize(1).anySatisfy(credentials -> {
  ...
}
)
```

Reply:
done